src/nvim/ops.c

```diff
@@ -3644,6 +3644,10 @@ int do_join(size_t count,
    */
   for (t = (linenr_T)count - 1;; t--) {
     cend -= currsize;
+	if(cend == NULL || curr == NULL){
```

Comment:
The intention of the instructions is to ensure the individual commits are still sane when doing a rebase, so that we don't break the ability to perform bisects later on.

So, if you're doing a rebase it's a good idea to insert `exec make -C build unittest` lines after every commit that's being kept.  Then when git processes the rebase, it will run `make -C build unittest` after every rebased commit.